spec/models/asset_spec.rb

```diff
@@ -129,7 +135,7 @@
 
     it "should handle empty name in mp3 tag" do
       asset = file_fixture_asset('japanese-characters.mp3', content_type: 'audio/mpeg')
-      expect(asset.permalink).to eq("01-oaee") # name is 01-\266Ե??\313"
+      expect(asset.permalink).to eq("01-¶ôμäèë") # name is 01-\266Ե??\313"
```

Comment:
This seems a bit strange! But then again, so did the original spec, this just seems stranger :)

Reply:
I removed the spec because its implementation is a duplicate now.